hieradata/class/backend.yaml

```diff
@@ -15,6 +15,12 @@ govuk::apps::share_sale_publisher::mongodb_nodes:
   - 'mongo-2.backend'
   - 'mongo-3.backend'
 
+govuk::apps:travel_advice_publisher::mongodb_name: 'govuk_content_production'
```

Comment:
Missing `:` here between `apps` and `travel_advice_publisher`, which is causing a build failure.